frontend/js/views/main.js

```diff
@@ -396,11 +398,18 @@ define(["jquery",
                         self.setLoadingProgress(60, i18next.t("startup.creating views"));
                         resolveView("player", annotationTool.playerAdapter);
                     }
+                    function failed() {
+                        // TODO Provide a translation
```

Comment:
This TODO is already done?

Reply:
Indeed. :sweat_smile: Took out the `TODO`.